pkgs/record_use/lib/record_use_internal.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
```

Comment:
This file seems to be used for anything but a single test. I think it can be deleted. The `pkgs/record_use/test/storage_test.dart` can just import any library it needs.

Reply:
It will be used by `native_assets_cli` to parse the `recorded_usages.json` at some point - but I can delete it for now.